packages/arb-avm-cpp/data_storage/src/arbstorage.cpp

```diff
@@ -90,3 +90,43 @@ std::unique_ptr<ReadTransaction> ArbStorage::makeReadTransaction() {
 std::unique_ptr<ReadWriteTransaction> ArbStorage::makeReadWriteTransaction() {
     return std::make_unique<ReadWriteTransaction>(datastorage);
 }
+
+rocksdb::Status resetDBExceptInbox(const std::string& dbpath,
+                                   const std::string& arbospath) {
```

Comment:
It's a bit weird to me that this is a standalone function instead of being a method on ArbStorage. It's possible the architecture makes that not possible, but I think it should be possible to make this a method that you have to call before initialize, which I think would be preferable. If this involves significant refactoring to change though, leave it be, it's only here for debugging/profiling.

Reply:
moved reset into initialize